### ISSUE TITLE
[Bug]: SSL Error when connecting to S3

### ISSUE THREAD

**Issue by tomh4** [coollabsio/coolify#4402]
### Error Message and Logs

`Unable to list contents for '', shallow listing

Reason: Error executing "ListObjectsV2" on "https://s3-backup.apps.servers.local/backup?list-type=2&prefix=&delimiter=%2F"; AWS HTTP error: cURL error 60: SSL certificate problem: unable to get local issuer certificate (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://s3-backup.apps.servers.local/backup?list-type=2&prefix=&delimiter=%2F`

### Steps to Reproduce

1. Create an S3 with coolify on a local server
2. Set up ssl and make everything work
3. Add root cert to coolify server
4. Add S3 storage 

### Coolify Version

v4.0.0-beta.372

### Are you using Coolify Cloud?

No (self-hosted)

### Operating System and Version (self-hosted)

Ubuntu 24.02 LTS

**Comment by tomh4** [comment]
My further investigations have show, that the host servers certificates are not available inside the coolify container, and there is also no option to add them.

Thus I would suggest to either map the host machines certificates by default, or give us an option inside the UI to add certificates to the coolify instance

**Comment by peaklabs-dev** [comment]
So you want to add an AWS S3 bucket with https, but you need your own SSL certificate for that to work correct?

**Comment by tomh4** [comment]
Yes @peaklabs-dev basically, the S3 is started by coolify as well so the cert should already be there... But even when I manually add the cert to the host machine of coolify it is not recognized. So I would imagine two scenarios that would be ok (one slightly better than the other):

- All Certs which are already used by apps managed with coolify should also be accepted or added to the coolify main instance
- Manually adding a cert chain to the host machine should also be available inside the coolify instance

**Comment by tomh4** [comment]
@peaklabs-dev will this be in a beta release as well or do we need to wait for 4.0.0 stable :) ?

**Comment by Noriller** [comment]
Stumbled on this.

I'm using an internal cert authority, the machines all have the needed root certs, but couldn't make it pass to coolify.

Tried to add as a volume in compose, but Suse handles certs a little differently than the expected.
